102.md

```diff
@@ -0,0 +1,53 @@
+NIP-102
+=======
+
+Live Activities
```

Comment:
@vitorpamplona can we rename this to make it more abstract?

would be nice to have something like https://schema.org/Event

Reply:
I just don't want to use Event because the basic building block of Nostr is an Event... So... But it should work for all events (arghhh)

Reply:
"Sessions" maybe.